lib/json/store.nit

```diff
@@ -111,7 +111,7 @@ class JsonStore
 
 	# Is there data are stored under `key`.
 	fun has_key(key: String): Bool do
-		return (store_dir / "{key}.json").file_exists
+		return ("{store_dir}/{key}.json".simplify_path).file_exists
```

Comment:
I'm not sure to understand how this fix paths. `simplify_path` does nothing signifiant (before and after paths are equivalent). and using a literal `\` in a string is fragile in border-line cases whereas the `/` operator know how to handle there cases.